pkg/anago/anago.go

```diff
@@ -279,6 +279,11 @@ func (s *Stage) Run() error {
 		return errors.Wrap(err, "generate changelog")
 	}
 
+	logger.WithStep().Info("Generating bill of materials")
+	if err := s.client.GenerateBillOfMaterials(); err != nil {
+		return errors.Wrap(err, "generating sbom")
+	}
```

Comment:
Out of curiosity: Will the changelog be part of the bill of materials?

Reply:
Not the content of the changelog itself, but we have to list the file as an item in the file